.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,34 @@
-> Please provide us with the following information:
-> ---------------------------------------------------------------
-
-### OS?
-> Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
-
-
+<!--
+IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
+-->
+### Bug Report or Feature Request (mark with an `x`)
+- [ ] bug report -> please search issues before submitting
+- [ ] feature request
+
+ <!-- if bug report, otherwise delete -->
 ### Versions.
-> Please run `ng --version`. If there's nothing outputted, please run in a Terminal: `node --version` and paste the result here:
+<!--
+Output from: `ng --version`.
+If nothing, output from: `node --version` and include OS details.
+  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
+-->
 
 
+ <!-- if bug report, otherwise delete -->
 ### Repro steps.
-> Was this an app that wasn't created using the CLI? What change did you do on your code? etc.
+<!-- Steps to reproduce this bug. -->
```

Comment:
Ok, I agree that the blurb I asked for is actually fairly intimidating and that's counterproductive.

But I would still like to see some elaboration of what's the kind of repro we're looking for. A lot (and I do mean a lot) of users just put in something like "When I run ng serve I get this error".

And that fits the bill of a simple repro... which is to run ng serve. But for us it's useless. It just results in a issue I mark as 'needs repro' and never have a lot of time to come back to.